{{cookiecutter.project_slug}}/.readthedocs.yml

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Add header comment with docs to this file format reference, like https://github.com/Ouranosinc/raven/blob/8798356b324339fc13e16a95878d8453615ececb/.readthedocs.yml#L1-L3